requirements.in

```diff
@@ -29,7 +29,7 @@ mysql-connector-python==8.0.26
 docker==4.4.4  # can't use >=5 because of tcconfig==0.26.0
 python-jenkins==1.7.0
 ssh2-python==1.0.0
-argus-alm==0.11.6
```

Comment:
@k0machi next time please include the full requirements.txt file, now we don't have the actual content of the image you created